.buildbot.sh

```diff
@@ -12,13 +12,13 @@ ulimit -d $((1024 * 1024 * 8)) # 8 GiB
     RUST_BACKTRACE=1 ./x.py test --config .buildbot.toml
 
 # Archive the build and put it in /opt
-TARBALL_TOPDIR=ykrustc-stage2
+TARBALL_TOPDIR=ykrustc-stage2 # To match [install] in the build config.
```

Comment:
This comment is a bit too cryptic, I think.

Reply:
It has to match the string used in the toml file:
```
[install]
prefix = ...
```

Reply:
(i'll push a better comment once it works)